src/reanimated2/layoutReanimation/web/transition/Linear.web.ts

```diff
@@ -0,0 +1,24 @@
+import { parseAnimationObjectToKeyframe } from '../animationParser';
+import type { TransitionData } from '../animationParser';
+
+export function LinearTransition(name: string, transitionData: TransitionData) {
+  const { translateX, translateY, scaleX, scaleY } = transitionData;
+
+  const linearTransition = {
+    name: name,
+    style: {
+      0: {
+        transform: [
+          {
+            translateX: `${translateX}px`,
+            translateY: `${translateY}px`,
+            scale: `${scaleX},${scaleY}`,
+          },
+        ],
+      },
+    },
+    duration: 300,
+  };
+
+  return parseAnimationObjectToKeyframe(linearTransition);
```

Comment:
What do you think about returning CSS keyframes string directly from here without calling `parseAnimationObjectToKeyframe`? 🤔

Reply:
At the beginning it was returning string. However, @piaskowyk said that it will be better to stay consistent with other animations and keep it as object. 

Also this opens up new possibilities. It will be much easier to modify object (for example to keep already existing transforms) than modify string.